tsconfig.json

```diff
@@ -34,7 +34,12 @@
       "@auth-service/*": ["src/app/service/auth/*"],
       "@store/*": ["src/app/store/*"],
       "@eco-news-images/*": ["src/assets/img/icon/econews/*"],
-      "@environment/*": ["src/environments/*"]
+      "@footer-images/*": ["src/assets/img/icon/footer/*"],
+      "@language-service/*": ["src/app/i18n/*"],
+      "@environment/*": ["src/environments/*"],
+      "@models/*": ["src/app/model/*"],
+      "@user-models/*": ["src/app/component/user/models/*"],
+      "@global-auth/*": ["src/app/component/auth/components/*"]
```

Comment:
The PR #446 made paths in alphabetical order -
![image](https://user-images.githubusercontent.com/6242270/89333539-c3b74980-d69d-11ea-899f-ceb246b4a19a.png)
- keep this convention

Reply:
fixed